agent/src/main/scala/za/co/absa/atum/agent/model/MeasuresBuilder.scala

```diff
@@ -16,34 +16,42 @@
 
 package za.co.absa.atum.agent.model
 
-import za.co.absa.atum.agent.exception.AtumAgentException.MeasureException
 import za.co.absa.atum.agent.model.AtumMeasure._
 import za.co.absa.atum.model.dto.MeasureDTO
+import org.apache.spark.internal.Logging
+
+import scala.util.Try
 
 /**
  * This object provides a functionality to convert a DTO representation of measures to the Agent's internal
  * representation of those objects.
  */
-private [agent] object MeasuresBuilder {
+private [agent] object MeasuresBuilder extends Logging {
 
   private [agent] def mapToMeasures(measures: Set[MeasureDTO]): Set[za.co.absa.atum.agent.model.AtumMeasure] = {
-    measures.map(createMeasure)
+    measures.flatMap { measure =>
+      createMeasure(measure) match {
+        case Some(value) =>
+          Some(value)
+        case None =>
+          logInfo(s"Measure not supported or unknown: $measure.")
```

Comment:
I'm thinking whether we want info or warning here? Might be beneficial to have it a bit more visible in logs perhaps?

Reply:
What do you think?

Reply:
I am fine with both really, changing to warn. then.